From 912081ad9cb70e88f20d0d58fb8d5fd2d617d82a Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 22 Aug 2015 22:18:22 -0700
Subject: [PATCH] Fix loading error if save files don't exist

This change in in accordance with the function's documentation, which
states that it should return true if the file is not found.
---
 mednafen/snes/interface.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mednafen/snes/interface.cpp b/mednafen/snes/interface.cpp
index 69b7b3b..0ac9fbb 100644
--- a/mednafen/snes/interface.cpp
+++ b/mednafen/snes/interface.cpp
@@ -344,7 +344,7 @@ static bool SaveMemorySub(bool load, const char *extension, SNES::MappedRAM *mem
     return(true);
 
    MDFN_PrintError(_("Error opening save file \"%s\": %s"), path.c_str(), ene.StrError());
-   return(false);
+   return(true);
   }
 
   if(memoryA && memoryA->size() != 0 && memoryA->size() != -1U)
-- 
1.9.5.msysgit.0

